### ISSUE TITLE
Do Now 4.03 - optional advanced exercise

### ISSUE THREAD

**Issue by normanbauer** [TEALS-IntroCS/2nd-semester-introduction-to-computer-science-principles#63]
As an optional exercise fro students that have completed the other exercises, have the students print out a pyramid of stars:

```
   *
  ***
******
```

---

---

**Comment by TEALS-IntroCS** [comment]
Good item for differentiation in the lab. The Do Now is meant to be quick.

**Comment by goldblatt** [comment]
cool added this as an extra bonus problem on 4.03, and called attention to this in the accommodation/differentiation

**Comment by normanbauer** [comment]
I could not find the note about using negative numbers to index from the back end of a list, so I added this comment here. When deleting from a list,  the method  I used a lot is to delete from the end so that index numbers are only shifted for list entries lower down in the list and index number for higher elements are not changed. This avoids the skipping over entries when indexing down from the top.  (Note: the indexing is zero based)

a = [1,2,3,4]
del a[2]
 print (a)
[1, 2, 4]
